docs/table_design/Data_distribution.md

```diff
@@ -112,51 +214,208 @@ CREATE TABLE site_access(
+
+  In the following example, the date range of all the partitions created at a time starts from 2021-01-01 and ends on 2021-01-04, with an incremental interval of one day:
+
+    ```SQL
+  CREATE TABLE site_access (
+      datekey DATE,
+      site_id INT,
+      city_code SMALLINT,
+      user_name VARCHAR(32),
+      pv BIGINT DEFAULT '0'
+  )
+  ENGINE=olap
+  DUPLICATE KEY(datekey, site_id, city_code, user_name)
+  PARTITION BY RANGE (datekey) (
+      START ("2021-01-01") END ("2021-01-04") EVERY (INTERVAL 1 DAY)
+  )
+  DISTRIBUTED BY HASH(site_id)
+  PROPERTIES ("replication_num" = "3" );
+    ```
+
+  It is equivalent to using the following `PARTITION BY` clause in the CREATE TABLE statement:
+
+    ```SQL
+  PARTITION BY RANGE (datekey) (
+  PARTITION p20210101 VALUES [('2021-01-01'), ('2021-01-02')),
+  PARTITION p20210102 VALUES [('2021-01-02'), ('2021-01-03')),
+  PARTITION p20210103 VALUES [('2021-01-03'), ('2021-01-04'))
+  )
+    ```
+
+- **Partition a table on a date-type column (DATE and DATETIME)  with different date intervals at the table creation**.
+
+  You can create batches of date partitions with different incremental interval by specifying different incremental interval in `EVERY` for each batch of partitions (make sure that the partition ranges between different batches do not overlap). Partitions in each batch are created according to the `START (xxx) END (xxx) EVERY (xxx)` clause. For example:
+
+    ```SQL
+  CREATE TABLE site_access(
+      datekey DATE,
+      site_id INT,
+      city_code SMALLINT,
+      user_name VARCHAR(32),
+      pv BIGINT DEFAULT '0'
+  )
+  ENGINE=olap
+  DUPLICATE KEY(datekey, site_id, city_code, user_name)
+  PARTITION BY RANGE (datekey) 
+  (
+      START ("2019-01-01") END ("2021-01-01") EVERY (INTERVAL 1 YEAR),
+      START ("2021-01-01") END ("2021-05-01") EVERY (INTERVAL 1 MONTH),
+      START ("2021-05-01") END ("2021-05-04") EVERY (INTERVAL 1 DAY)
+  )
+  DISTRIBUTED BY HASH(site_id)
+  PROPERTIES(
+      "replication_num" = "3"
+  );
+    ```
+
+  It is equivalent to using the following `PARTITION BY` clause in the CREATE TABLE statement:
+
+    ```SQL
+  PARTITION BY RANGE (datekey) (
+  PARTITION p2019 VALUES [('2019-01-01'), ('2020-01-01')),
+  PARTITION p2020 VALUES [('2020-01-01'), ('2021-01-01')),
+  PARTITION p202101 VALUES [('2021-01-01'), ('2021-02-01')),
+  PARTITION p202102 VALUES [('2021-02-01'), ('2021-03-01')),
+  PARTITION p202103 VALUES [('2021-03-01'), ('2021-04-01')),
+  PARTITION p202104 VALUES [('2021-04-01'), ('2021-05-01')),
+  PARTITION p20210501 VALUES [('2021-05-01'), ('2021-05-02')),
+  PARTITION p20210502 VALUES [('2021-05-02'), ('2021-05-03')),
+  PARTITION p20210503 VALUES [('2021-05-03'), ('2021-05-04'))
+  )
+    ```
+
+- **Partition a table on an integer type column at the table creation**
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/a7dfe7d7-6a68-4658-8917-f05a3bf0b27b)
Add a hyphen (-) between "integer" and "type", and remove "the"